airbnb_project/listings/harvester_app/harvester/spiders/listings_spider.py

```diff
@@ -237,8 +212,8 @@ def _extract_listing_data(self, result: Dict[str, Any]) -> Dict[str, Any] | None
         listing = result.get("listing", {})
         listing_id: str = listing.get("id")
 
-        if not listing_id or listing_id in self.ALREADY_SCRAPED_LISTINGS:
-            print(f"Duplicate or missing listing ID: {listing_id}")
+        if not listing_id:
+            print(f"Missing listing ID: {listing_id}")
```

Comment:
if the `listing_id` is `None` - I'm not sure if we'll get any value by logging `None`. Is there anything else we can log?

Reply:
As previously mentioned, the url the id is scarped from can be one, I will look into other things to log.